pages/documentation/migration-V3.md

```diff
@@ -770,75 +764,94 @@ These instructions will help you update your `@import` references to the new syn
 - // components import needs to be last
 - @forward "uswds-theme-components";
 + @forward "uswds-theme";
-@forward "uswds";
-@forward "uswds-custom-styles";
+  @forward "uswds";
+  @forward "uswds-custom-styles";
 ```
 
-Now your project is using its theme settings in the proper USWDS 3.0 format! Now you can remove the old theme files from your project.
+Now your project is using its theme settings in the proper USWDS 3.0 format! You can safely remove the old theme files from your project.
 
 #### Use "uswds-core" for any custom USWDS Sass
```

Comment:
Reformatted the USWDS-core section and Optimization sections to include and highlight steps for accomplishing the task

Reply:
Agreed